backend/db/migrations/V1.0.38__update_built_in_announcement.sql

```diff
@@ -1,5 +1,5 @@
 SET search_path TO pay_transparency;
 
 -- add a 'published_on' date to the built-in announcement
-update announcement set published_on = LOCALTIMESTAMP AT TIME ZONE 'UTC-7' 
+update announcement set published_on = LOCALTIMESTAMP AT TIME ZONE 'UTC' 
```

Comment:
I don't think modifying this file will work when deploying to Dev. The dev database is already v1.0.38 and doesn't have the ability to undo a previous update and then apply this update anew. I think you'll have to create a new version for this change.

Reply:
Fixed.